petclinic/actors/customers/Makefile

```diff
@@ -13,3 +13,6 @@ CALL_ALIAS = petclinic/customers
 
 include ./actor.mk
 
+link-sqldb:
+	wash ctl link put $(shell make --silent actor_id) \
+	$(SQLDB_ID) $(shell base64 -w0 sql-linkdef.json)
```

Comment:
I know I used this in the todo example. I think you'll need the last part to be
`config_b64=$(shell base64 -w0 sql-linkdef.json)`